ProjectTemplates/gmodnet-module/TemplateModule.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>Library</OutputType>
+		<TargetFramework>net5.0</TargetFramework>
+		<LangVersion>9.0</LangVersion>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="GmodNET.API" Version="0.7.0-beta.2.28828564.master" />
```

Comment:
Fixing on some particular version of nighty build is suboptimal. Can we try wildcard `0.7.0-*`?

Reply:
resolved in https://github.com/GmodNET/Templates/pull/2/commits/b35fbb5c2afa30db85c220f50bde175673437d7c